react/features/speaker-stats/components/TimeElapsed.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+import {
+    getHoursCount,
+    getMinutesCount,
+    getSecondsCount
+} from '../../base/util/timeUtils';
+
+/**
+ * React component for displaying total time elapsed. Converts a total count of
+ * milliseconds into a more humanized form: "# hours, # minutes, # seconds".
+ * With a time of 0, "0 seconds" will be returned.
+ *
+ * @extends Component
+ */
+class TimeElapsed extends Component {
```

Comment:
Why hand-roll this instead of using something like: https://momentjs.com/docs/#/durations/humanize/